index.html

```diff
@@ -1151,6 +1176,160 @@ <h3>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Icons
+      </h2>
+      <p>
+        Icons can be used by the user agent in multiple places, such as on a
+        home-screen, in an app store, or even on top of notifications.
```

Comment:
Nit: perhaps the notifications example isn't needed? The "even" seems a bit much in any case.

Reply:
Rephased